app/controllers/course/assessment/submission/submissions_controller.rb

```diff
@@ -101,6 +102,8 @@ def check_password
     return if !@assessment.password_protected? || can?(:manage, @assessment)
 
     unless authentication_service.authenticated?
+      log_service.log_submission_access(request)
```

Comment:
If failed attempt, the 2 session IDs will be different or empty. If new submission or correct password, the 2 session IDs will be the same. Can look at the screenshot for examples. Maybe we can add another column that will make it more obvious?

When user inputs password correctly, that one is logged here https://github.com/Coursemology/coursemology2/pull/2145/files#diff-d33f0d7e0efb38504f82d4b17c98d3f3R37